### PR TITLE
Feature/17250 select deselect

### PR DESCRIPTION
**Pull request by pris54**
Note: awaiting UX review of existing features to confirm which ones need this refactoring.

### CODE REVIEW

File: admin/assets/javascripts/global-scripts.js [hunk 1]
```diff
@@ -3,190 +3,208 @@
...
       var el = $('.multiple-choice-mtc > input:checkbox:checked')
-      if (el.length > 0) {
-        enableButtonStatus(true)
-      } else {
-        enableButtonStatus(false)
-      }
+      return (el.length > 0)
+    },
+    validateForm: function () {
+      return (pupilsNotTakingCheck.isRadioChecked() && pupilsNotTakingCheck.isCheckboxChecked())
```
Comment (GeorgeChatzigiannis) [inline admin/assets/javascripts/global-scripts.js]: dont need the brackets

File: admin/assets/javascripts/global-scripts.js [hunk 2]
```diff
@@ -3,190 +3,208 @@
...
+     * @returns {boolean}
      */
-    var countCheckboxes = function () {
-      var el = $('.multiple-choice-mtc > input:checkbox:checked')
-      $('#totalPupilsSelected').text(el.length)
-    }
-
+    isRadioChecked: function () {
+      var el = $('input:radio[name="attendanceCode"]:checked')
+      return (el.length > 0)
```
Comment (GeorgeChatzigiannis) [inline admin/assets/javascripts/global-scripts.js]: here as well

File: admin/assets/javascripts/global-scripts.js [hunk 3]
```diff
@@ -3,190 +3,208 @@
...
-      if (el.length > 0) {
-        $('#stickyBanner').addClass('show')
-      } else {
-        $('#stickyBanner').removeClass('show')
-      }
-    })
+  var generatePins = {
+    isCheckboxChecked: function () {
+      var el = $('.multiple-choice-mtc > input:checkbox:checked')
+      return (el.length > 0)
```
Comment (GeorgeChatzigiannis) [inline admin/assets/javascripts/global-scripts.js]: here too

File: admin/assets/javascripts/global-scripts.js [hunk 4]
```diff
@@ -3,190 +3,208 @@
...
      */
-    var isCheckboxChecked = function () {
+    isCheckboxChecked: function () {
       var el = $('.multiple-choice-mtc > input:checkbox:checked')
-      if (el.length > 0) {
-        enableButtonStatus(true)
-      } else {
-        enableButtonStatus(false)
-      }
+      return (el.length > 0)
```
Comment (GeorgeChatzigiannis) [inline admin/assets/javascripts/global-scripts.js]: another one

File: admin/assets/javascripts/global-scripts.js (hunk 1 above)
Comment (pris54) [reply admin/assets/javascripts/global-scripts.js]: @GeorgeChatzigiannis I'm aware of alternative formats, I find the brackets useful to make the code more readable, especially with a two character indentation.

Comment (pris54) [reply admin/assets/javascripts/global-scripts.js]: @GeorgeChatzigiannis I'll remove the parenthesis just for the sake of get this PR approved TODAY. Thanks.

**Pull request merged**